modules/configuring-a-workload-to-use-shared-cpus.adoc

```diff
@@ -0,0 +1,130 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="configuring-a-workload-to-use-shared-cpus_{context}"]
+= Configuring a workload to use shared CPUs
+
+You can pin lightweight workload tasks to shared CPUs to improve CPU resource efficiency. When you enable shared CPUs on a node, containers deployed on the node feature the `OPENSHIFT_SHARED_CPUS` and `OPENSHIFT_ISOLATED_CPUS` environment variables. These variables show the IDs of the shared and isolated CPUs. 
+
+You can use the `OPENSHIFT_SHARED_CPUS` environment variable to pin latency-tolerant application threads to shared CPUs. 
+
+For latency-sentsitive application threads, use the `OPENSHIFT_ISOLATED_CPUS` environment variable.
+
+.Prerequisites
+
+* Log in as a user with `cluster-admin` privileges.
+
+* You defined and enabled shared CPUs by using a performance profile.
+
+//* You enabled the `TechPreviewNoUpgrade` feature set on the cluster. For more information about enabling a feature set, see the _Additional resources_ section.
+// +
+// [WARNING]
+// ====
+// Enabling the `TechPreviewNoUpgrade` feature set cannot be undone and prevents minor version updates. These feature sets are not recommended on production clusters.
+// ====
+
+.Procedure
+
+. Create a namespace by running the following command.
++
+[source,terminal]
+----
+$ oc create namespace <namespace_name>
+----
++
+[NOTE]
+====
+To use shared CPUs, you must use a namespace with the `workload.mixedcpus.openshift.io/allowed` annotation.
+====
+
+. Add the required annotation to the namespace by running the following command:
++
+[source,terminal]
+----
+$ oc annotate namespace <namespace_name> workload.mixedcpus.openshift.io/allowed=''
+----
++
+.Example output
+[source,terminal]
+----
+namespace/<namespace_name> annotate
+----
+
+. Create a `Pod` resource that requests shared CPUs:
+
+.. Create a YAML file that defines the `Pod` resource:
++
+--
+.Example `shared-cpu-pod.yaml` file
+[source,yaml]
+----
+kind: Pod
```

Comment:
Another constrain that was added is that pod must be running withing a namespace with special annotation:
```
kind: Namespace
Metadata:
  name: mynamespace
  annotations:
    workload.mixedcpus.openshift.io/allowed: "" 
```

Reply:
And then the pod should be:
```
kind: Pod
metadata: 
  namespace: mynamespace
  name: dpdk-pod
...
```

Reply:
This should already be specified [here](https://github.com/openshift/openshift-docs/pull/68393/files/98e68334618d6ac72c6a07e47b4057195adda39b#diff-fe8e4af6468e622bbc6f05e483b8a89083a73d304ff97cdb59cdf97512d37e33R34-R43).

And [here](https://github.com/openshift/openshift-docs/pull/68393/files/98e68334618d6ac72c6a07e47b4057195adda39b#diff-fe8e4af6468e622bbc6f05e483b8a89083a73d304ff97cdb59cdf97512d37e33R78). 

Is this OK or am I missing something?

Reply:
The links are broken

Reply:
Ah sorry I updated the PR so the line range broke. I meant the [steps](https://github.com/openshift/openshift-docs/pull/68393/files#diff-fe8e4af6468e622bbc6f05e483b8a89083a73d304ff97cdb59cdf97512d37e33R39-R50) earlier in this procedure mention that you need to add the annotation. And a callout for the Pod resource metnions to use the namespace with the annotation.